### DISCUSSION TITLE
How to solve the periods of Pnew/2, Pnew, and 2Pnew calculate by lomb-scargle? Can the geometry estimator apply in contact binaries?

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#808] [Questions]
![geometry estimator](https://github.com/phoebe-project/phoebe2/assets/32772701/aac85efe-c2e6-444d-96ca-cd3ef5b60c30)
![geometry estimator](https://github.com/phoebe-project/phoebe2/assets/32772701/fe7c0c0c-dd47-415c-aaa3-11f12d7380f1)

**Comment by kecnry** [comment]
The geometry estimator takes a phased light curve as input, so it sounds like the authors manually set the orbital period based on the returned BLS period and ran the geometry estimator for each of those three input phased light curves - but you would need to ask them for more specific details on how they compared the results (from this snippet, it's not clear if they ran a forward model immediately after or did any further optimization on any parameters).

> Can the geometry estimator apply in contact binaries?

No, the geometry estimator is not supported for contact systems or single stars.
